terraform/modules/concourse_web/asg.tf

```diff
@@ -24,6 +24,9 @@ resource "aws_autoscaling_group" "web" {
 
   lifecycle {
     create_before_destroy = true
+    ignore_changes = [
+      max_size
```

Comment:
Is this for the scheduled scaling each day?

Reply:
This stops it constantly making changes to the ASG if we deploy during the day.  It's all automated by the day and night jobs and shouldn't be effected by this.